config/example_indexed.json

```diff
@@ -0,0 +1,41 @@
+{
+	"type": "indexed",
+	"dimensionality": 3,
+	"chunks": [
+		1000,
+		1000,
+		1000
+	],
+	"dataset": {
+		"filepath": "path_to_file",
```

Comment:
Will need to be removed as I forgot to remove the filepath key from the indexed config. I have removed it in the BODCDMSYS-3652 branch earlier today, but it might just be best just manually removing it rather than doing a merge.

Reply:
Done :)